### PR TITLE
Experimentally enable prometheus in gce 5k performance tests

### PR DESCRIPTION
**Pull request by mm4tt**
https://github.com/kubernetes/kubernetes/issues/74213

### CODE REVIEW

File: config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml [hunk 1]
```diff
@@ -386,6 +386,8 @@ periodics:
       - --test-cmd-args=--report-dir=/workspace/_artifacts
       - --test-cmd-args=--testconfig=testing/density/config.yaml
       - --test-cmd-args=--testconfig=testing/load/config.yaml
+      # TODO(https://github.com/kubernetes/kubernetes/issues/74213): Set everywhere if it works
+      - --test-cmd-args=--enable-prometheus-server
```
Comment (wojtek-t) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml]: Discussed offline - the current setup will not work with kubemark.

Comment (mm4tt) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml]: Done.

**Pull request merged**